src/test/operators/delete_test.cpp

```diff
@@ -137,6 +137,35 @@ TEST_F(OperatorsDeleteTest, DetectDirtyWrite) {
   EXPECT_TABLE_EQ_UNORDERED(validate->get_output(), expected_result->get_output());
 }
 
+TEST_F(OperatorsDeleteTest, EmptyDelete) {
+  auto tx_context = TransactionManager::get().new_transaction_context();
+
+  auto table_scan = create_table_scan(_gt, ColumnID{0}, PredicateCondition::Equals, "112233");
+
+  table_scan->execute();
+
+  EXPECT_EQ(table_scan->get_output()->chunk_count(), 0u);
+
+  auto delete_op = std::make_shared<Delete>(_table_name, table_scan);
+  delete_op->set_transaction_context(tx_context);
+
+  delete_op->execute();
+
+  EXPECT_FALSE(delete_op->execute_failed());
+
+  // MVCC commit.
+  tx_context->commit();
+
+  // Get validated table which should be the original one
+  auto t_context = TransactionManager::get().new_transaction_context();
```

Comment:
one variable named "tx_context" and another named "t_context"? :P

Reply:
The virtues of copy and paste ;)